docs/source/x_ext.rst

```diff
@@ -531,13 +503,83 @@ The |processor| shall cause an illegal instruction fault when attempting to exec
 A |coprocessor| can (only) accept an offloaded instruction when:
 
 * It can handle the instruction (based on decoding ``instr``).
-* The required source registers are marked valid by the offloading core  (``issue_valid`` is 1 and required bit(s) ``rs_valid`` are 1).
+* There are no structural hazards that would prevent execution
 
 A transaction is considered offloaded/accepted on the positive edge of ``clk`` when ``issue_valid``, ``issue_ready`` are asserted and ``accept`` is 1.
 A transaction is considered not offloaded/rejected on the positive edge of ``clk`` when ``issue_valid`` and ``issue_ready`` are asserted while ``accept`` is 0.
 
 The signals in ``issue_resp`` are valid when ``issue_valid`` and ``issue_ready`` are both 1. There are no stability requirements.
 
+Register interface
+~~~~~~~~~~~~~~~~~~
+:numref:`Register interface signals` describes the register interface signals.
+
+.. table:: Register interface signals
+  :name: Register interface signals
+
+  +---------------------------+-----------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | **Signal**                | **Type**        | **Direction**   | **Description**                                                                                                              |
+  |                           |                 | (|processor|)   |                                                                                                                              |
+  +---------------------------+-----------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``register_valid``        | logic           | output          | Register request valid. Indicates that |processor| wants to offload an instruction.                                             |
```

Comment:
Description not accurate (copy / paste from issue_valid)

Reply:
fixed